src/ImageProcessorCore/Formats/Tiff/TiffExifExtractor.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace ImageProcessorCore.Formats
+{
+    /// <summary>
+    ///
+    /// </summary>
+    internal class TiffExifExtractor : ITiffVisitor
+    {
+        private readonly Dictionary<ushort, TiffTag> _exifTags; 
+
+        public List<ImageProperty> Properties { get; private set; }
+
+        public TiffExifExtractor()
+        {
+            Properties = new List<ImageProperty>();
+            
+            _exifTags = TiffTagRegistry.Instance.Tags
+                .Where( i => i.TagGroup == "Exif")
+                .ToDictionary(i => i.TagId);
+
+        }
+
+        public void Visit(TiffProperty property)
+        {
+            if (_exifTags.ContainsKey(property.Tag.TagId))
+            {
+                // until the image property can hold more than just a string value....
+                ImageProperty imageProperty = new ImageProperty(property.Tag.Name, property.Value.ToString());
```

Comment:
Feel free to change `ImageProperty `to suit your needs. I'm assuming here we'd add make it's value an `object`and add a `Type` property which returns whether it's a `string`, `Rational`etc.